github-pullrequest-plugin/src/main/java/org/jenkinsci/plugins/github/pullrequest/trigger/JobRunnerForCause.java

```diff
@@ -83,7 +83,7 @@ public boolean apply(GitHubPRCause cause) {
                 trigger.getRemoteRepo()
                         .createCommitStatus(cause.getHeadSha(),
                                 GHCommitState.PENDING,
-                                null,
+                                job.getAbsoluteUrl(),
```

Comment:
Yes.

Sent from my iPhone

> On 4. 4. 2016, at 18:34, Kanstantsin Shautsou <notifications@github.com> wrote:
> 
> In github-pullrequest-plugin/src/main/java/org/jenkinsci/plugins/github/pullrequest/trigger/JobRunnerForCause.java:
> 
> > @@ -83,7 +83,7 @@ public boolean apply(GitHubPRCause cause) {
> >                  trigger.getRemoteRepo()
> >                          .createCommitStatus(cause.getHeadSha(),
> >                                  GHCommitState.PENDING,
> > -                                null,
> > +                                job.getAbsoluteUrl(),
> is it for prestatus?
> 
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
>